src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -860,6 +864,55 @@ private static void VerifyRefAssemblyClient(string lib_cs, string source, Action
         }
 
         [Theory]
+        [InlineData("")]
+        [InlineData(@"[assembly: System.Reflection.AssemblyVersion(""1"")]")]
+        [InlineData(@"[assembly: System.Reflection.AssemblyVersion(""1.0.*"")]")]
```

Comment:
Thanks for the clarification.
I see that VersionHelper.GenerateVersionFromPatternAndCurrentTime replaces the wildcard (maxValue) with the timestamp of the build.
That means if you use a wildcard, the ref assembly will not be deterministic. I wonder if I should switch the compilation to be deterministic if either /refonly or /refout is used.